docs/components/modeler/desktop-modeler/troubleshooting.md

```diff
@@ -0,0 +1,100 @@
+---
+id: troubleshooting
+title: Fix common issues with the Desktop Modeler
+description: "This page lists common issues with the Desktop Modeler and potential resolutions."
+---
+
+## How to start the Modeler without plug-ins?
+
+You can start the modeler with the [`--disable-plugins` flag](https://docs.camunda.io/docs/components/modeler/desktop-modeler/flags/#disable-plug-ins).
+
+## How to obtain the Modeler logs?
+
+Depending on your operating system you can find the Modeler logs in different places:
+
+#### Windows
+
+```plain
+%APPDATA%\Camunda Modeler\logs
+```
+
+#### Mac OS
+
+```plain
+~/Library/Logs/Camunda Modeler
+```
+
+#### Linux
+
+```plain
+~/.config/camunda-modeler/logs
+```
+
+## I cannot connect to Zeebe
+
+> I try to connect to deploy to a remote Zeebe instance and the modeler tells me "cannot find a running Zeebe".
+
+Check if you can connect to Zeebe through another client, i.e. [`zbctl`](https://docs.camunda.io/docs/apis-tools/cli-client/). If that works [further debug your Zeebe connection](#debug-zeebe-connection-issues), if not try to resolve that [general connection issue](#resolve-general-zeebe-connection-issue).
+
+## Resolve general Zeebe connection issue
+
+> I'm trying to connect to Zeebe from both Desktop modeler _and_ [`zbctl`](https://docs.camunda.io/docs/apis-tools/cli-client/) to a running Zeebe cluster and non of them works.
+
+This can have multiple reasons:
+
+- The remote endpoint is not reachable
+- The connection is passed through a proxy :arrow_right: Ensure the proxy supports [HTTP/2 transport forwarding for GRPC](https://docs.camunda.io/docs/self-managed/platform-deployment/troubleshooting/#zeebe-ingress-grpc)
+
+## Debug Zeebe connection issues
```

Comment:
Add fragment about environment variables usage: `GRPC_VERBOSITY=DEBUG GRPC_TRACE=all`
This allows to understand the issues better.

Reply:
Done via https://github.com/camunda/camunda-platform-docs/pull/2091/commits/91513b3259495da3c180742ecee9317d998218c8.